Sources/PathKit.swift

```diff
@@ -152,8 +152,18 @@ extension Path {
   ///
   public func abbreviate() -> Path {
 #if os(Linux)
-    // TODO: actually de-normalize the path
-    return self
+    let env = ProcessInfo.processInfo.environment
+    guard let home = env["HOME"], !home.isEmpty else { return self }
+    guard let homeRange = self.path.range(of: home, options: [.anchored, .caseInsensitive]) else { return self }
+    let withoutHome = Path(self.path.replacingCharacters(in: homeRange, with: ""))
+    
+    if withoutHome.path.isEmpty || withoutHome.path == "/" {
```

Comment:
Yes, I just checked and Swift itself only has macOS and Linux CI. So there would easily be breakages in Swift itself on these platforms.